src/taco-utils/tacoErrorCodes.ts

```diff
@@ -18,6 +18,9 @@ module TacoUtility {
         PackageLoaderNpmUpdateFailedEaccess = 206,
         PackageLoaderNpmUpdateFailedWithCode = 207,
         PackageLoaderUpdateUnableToRecover = 208,
+        PackageLoaderRunPackageDoesntExportBinary = 209,
+        PackageLoaderRunPackageDoesntHaveRequestedBinary = 210,
+        PackageLoaderRunPackageBadBinaryFilePath = 211,
```

Comment:
Do these new error codes have associated strings in taco-utils\resources\en\resources.json? If so, they do not seem to be part of this PR :)